cmake/FindLuaJit.cmake

```diff
@@ -32,8 +32,11 @@ if(LUAJIT_USE_STATIC)
 endif()
 
 list(APPEND LUAJIT_NAMES luajit-5.1)
+if(MSVC)
+  list(APPEND LUAJIT_NAMES lua51)
```

Comment:
cc @equalsraf. All I know is that without this MSVC doesn't find the library. As for why it's that why I have not investigated.